server.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express')
+
+const app = express()
+const PORT = 3000
```

Comment:
Здесь можно порт указать таким образом:
```suggestion
const PORT = process.env.PORT || 3000; 
```
Тогда будет возможность брать значение из env-переменных, а уже если там её не окажется, то брать значение по умолчанию